lib/iris/aux_factory.py

```diff
@@ -444,9 +420,8 @@ def calc_bounds():
                                   'These are being disregarded.',
                                   UserWarning, stacklevel=2)
                     orography_pts = nd_points_by_key['orography']
-                    orography_pts_shape = list(orography_pts.shape)
-                    orography = orography_pts.reshape(
-                        orography_pts_shape.append(1))
+                    bds_shape = list(orography_pts.shape) + [1]
+                    orography = orography_pts.reshape(bds_shape)
```

Comment:
@pp-mo The comment on [line 408](https://github.com/SciTools/iris/pull/2604/files#diff-42d68d7f5fd99ece9d3d591cd777d86cR408) is no longer relevant. We don't have closures anymore now that we've purged the `_LazyArray` implementation ... could you please remove all such `closure` related comments, as they are no longer correct, thanks!